dev/radar/appsettings.json

```diff
@@ -0,0 +1,154 @@
+    },
+    "Clin": {
+      "Connection": "LEAF_CLIN_DB",
+      "DefaultTimeout": 180,
+      "RDBMS":  "MARIADB",
+      "Cohort": {
+        "QueryStrategy": "PARALLEL",
+        "MaxParallelThreads": 5
+      }
+    }
+  },
+  "Authentication": {
+    "Mechanism": "UNSECURED",
+    "SessionTimeoutMinutes": 480,
+    "InactivityTimeoutMinutes": 20,
+    "Logout": {
+      "Enabled": true,
+      "URI": "https://www.github.com"
+    },
+      "SAML2": {
+        "Headers": {
+          "ScopedIdentity": "eppn"
+        }
+      }
+    },
+  "Authorization": {
+    "Mechanism": "UNSECURED",
+    "AllowAllAuthenticatedUsers": true,
+    "UnsecuredIsAdmin": true,
+    "SAML2": {
+      "HeadersMapping": {
+        "Entitlements": {
+          "Name": "gws-groups",
+          "Delimiter": ";"
+        }
+      },
+      "RolesMapping": {
+        "User": "urn:mace:users",
+        "Super": "urn:mace:supers",
+        "Identified": "urn:mace:phi",
+        "Admin": "urn:mace:sudos",
+        "Federated": "urn:mace:federated"
+      }
+    }
+  },
+  "Attestation": {
+    "Enabled": true
+  },
+  "Compiler": {
+    "Alias": "@",
+    "FieldPersonId": "PatientId",
+    "FieldEncounterId": "visit_occurrence_id"
+  },
+  "Cohort": {
+    "RowLimit": 200000,
+    "ExportLimit": 500
+  },
+  "Export": {
+    "REDCap": {
+      "Enabled": false,
+      "ApiURI": "https://redcap.example.edu/api/",
+      "BatchSize": 10,
+      "RowLimit": 5000,
+      "Scope": "washington.edu",
+      "SuperToken": "LEAF_REDCAP_SUPERTOKEN",
+      "IncludeScopeInUsername": true
+    },
+    "CSV": {
+      "Enabled": true
+    }
+  },
+  "Import": {
+    "REDCap": {
+      "Enabled": false,
+      "ApiURI": "https://redcap.example.edu/api/",
+      "BatchSize": 1000
+    }
+  },
+  "Deidentification": {
+    "Patient": {
+      "Enabled": false,
+      "DateShifting": {
+        "Increment": "DAY",
+        "LowerBound": -365,
+        "UpperBound": 365
+      }
+    },
+    "Cohort": {
+      "Enabled": true,
+      "Noise": {
+        "Enabled": false,
+        "LowerBound": -10,
+        "UpperBound": 10
+      },
+      "LowCellSizeMasking": {
+        "Enabled": false,
+        "Threshold": 10
+      }
+    }
+  },
+  "Client": {
+    "Map": {
+      "Enabled": true,
+      "TileURI": "https://server.arcgisonline.com/ArcGIS/rest/services/World_Street_Map/MapServer/tile/{z}/{y}/{x}"
+    },
+    "Visualize": {
+      "Enabled": true,
+      "ShowFederated": false
+    },
+    "Timelines": {
+      "Enabled":  true
+    },
+    "PatientList": {
+      "Enabled": true
+    },
+    "Help": {
+      "Enabled": true,
+      "Email": "leaf-support@example.edu",
+      "URI": "https://www.example.edu/leaf-faq"
+    }
+  },
+  "Notification": {
+    "Enabled": false,
+    "Email": {
+      "Server": "server",
+      "UseSSL": true,
+      "Sender": {
+        "Address": "sender@u.edu"
+      },
+      "Receiver": {
+        "Address": "receiver@u.edu"
```

Comment:
I think it's OK to leave these as-is for now, but in the not-so-distant future we probably want to request a shared NetID to receive or send mail for help requests and to handle notifications at some later date.

Reply:
We could point this at cirg-adm+leaf@uw.edu, as a short term thing or for testing.

Reply:
> We could point this at [cirg-adm+leaf@uw.edu](mailto:cirg-adm+leaf@uw.edu), as a short term thing or for testing.

Great idea.

Reply:
Done! (see 4859ba37cfb095aa4f4af1dade042ce6c288f674)

We'll also need to create a mailbox on our mailserver and configure the username/password before this is functional